### DISCUSSION TITLE
IANA port assignment

### DISCUSSION THREAD

**Discussion by elliotblackburn** [etsy/statsd#739] [Team Posts]
Hi folks, I just wanted to highlight https://github.com/statsd/statsd/issues/682 to anyone interested. If we did go ahead with this, we would need an "assignee email" to be used on behalf of the statsd community. Do we currently have a domain name that we might wish to use? It's not a requirement, I could just put my email (or someone elses in) for now and we could get it changed at a later date, but it would seem sensible to use a domain we could have some sort of shared access / email forwarding setup for.

**Comment by mrtazz** [comment]
There is currently no domain name we have registered for statsd. In addition statsd.org seems to be squatted and directing to a potential malware/phishing site. I think having a domain name would be useful, but if it's too much work (realistically I can't take on more admin work right now), having a personal email be registered for now and changing it later if we have a domain also sounds like a good plan.

**Comment by mheffner** [comment]
So I just registered statsd.app, most of the other statsd TLD's are taken (ie. squatted). If we want to use that, I can setup an email list under this, like maintainers@statsd.app or something. Any suggestions for free email services for open source projects?

**Comment by elliotblackburn** [comment]
I have an email provider that allows me attach multiple domains / inboxes for no extra cost. So I could spin up a mailbox for it under that account for free (it's charged based on daily outgoing emails, so as long as no one sends over 200 emails a day we'd be good). First though, I'll fire off an email to fastmail and some similar services to see if they offer some sort of open source free tier.

**Comment by elliotblackburn** [comment]
Fastmail just got back to me and seem keen, turns out they use statsd themselves. They've asked a few questions about our usage which I've answered, hopefully they'll reply soon with whether or not they can offer us something for free.

**Comment by elliotblackburn** [comment]
@mheffner Fastmail has just given us 3 years free, do you have an email address I can contact you on to coordinate getting your account setup and getting the domain name hooked up?

**Comment by mheffner** [comment]
@BlueHatbRit yeah, send to `mikeh@fesnel.com`

**Comment by mheffner** [comment]
Alright, we should be good to go with `maintainers@statsd.app`. Thanks @BlueHatbRit for setting up Fastmail. I've setup my account to just forward to my existing gmail.

I realized while setting up the DNS records that Google might provide free email forwarding as a basic option for domains, without having to pay for a gsuite acct. If we have issues with Fastmail that could be an option.

**Comment by elliotblackburn** [comment]
Oh well, at least we have a backup plan! Thanks for sorting that out, I'll get cracking on the application over this week and maybe submit a markdown version to the repo for review.

**Comment by mrtazz** [comment]
Thanks for doing all this! I came back from vacation recently and was excited to see this happen! :)

**Comment by elliotblackburn** [comment]
It's probably worth noting here for the record as well that, sadly, our application was rejected (https://github.com/statsd/statsd/pull/684#issuecomment-536482900) however SELinux and Redhat seem to be including our port none the less which is great news 🚀
